ROADMAP.md

```diff
@@ -8,21 +8,21 @@ We hope that the items listed below will inspire further engagement from the com
 Any dates listed below and the specific issues that will ship in a given milestone are subject to change but should give a general idea of what we are planning.
 See the [GitHub project boards](https://github.com/rook/rook/projects) for the most up-to-date issues and their status.
 
-## Rook Ceph 1.13
+## Rook Ceph 1.14
 
-The following high level features are targeted for Rook v1.13 (November 2023). For more detailed project tracking see the [v1.13 board](https://github.com/rook/rook/projects/30).
+The following high level features are targeted for Rook v1.14 (April 2024). For more detailed project tracking see the [v1.14 board](https://github.com/rook/rook/projects/31).
 
-* OSD encryption on partitions [#10984](https://github.com/rook/rook/issues/10984)
+* Allow setting the application name on a CephBlockPool [#13744](https://github.com/rook/rook/pull/13744)
 * Object Store
   * Pool sharing for clusters where many object stores are required [#11411](https://github.com/rook/rook/issues/11411)
-* CephFS
-  * Automatic subvolume group pinning [#12607](https://github.com/rook/rook/issues/12607)
-* Ceph-CSI [v3.10](https://github.com/ceph/ceph-csi/issues?q=is%3Aopen+is%3Aissue+milestone%3Arelease-v3.10.0)
+* Support replacing a single OSD when a metadataDevice is configured with multiple OSDs [#13240](https://github.com/rook/rook/issues/13240)
+* Option to run all components on the host network [#13571](https://github.com/rook/rook/issues/13571)
+* Separate CSI image repository and tag for all images in the helm chart [#13585](https://github.com/rook/rook/issues/13585)
+* Ceph-CSI [v3.11](https://github.com/ceph/ceph-csi/issues?q=is%3Aopen+is%3Aissue+milestone%3Arelease-v3.11.0)
+* Add build support for Go 1.22 [#13738](https://github.com/rook/rook/pull/13738)
 
 ## Kubectl Plugin
 
-Features are planned in the 1.13 time frame for the [Kubectl Plugin](https://github.com/rook/kubectl-rook-ceph).
-  * Recover the CephCluster CR after accidental deletion [#68](https://github.com/rook/kubectl-rook-ceph/issues/68)
-  * Force cleanup the cluster if graceful uninstall is not desired [#131](https://github.com/rook/kubectl-rook-ceph/issues/131)
-  * Provide a restricted set of commands based on a build flag [#174](https://github.com/rook/kubectl-rook-ceph/issues/174)
-  * Collect details to help troubleshoot the csi driver [#69](https://github.com/rook/kubectl-rook-ceph/issues/69)
+Features are planned in the 1.14 time frame for the [Kubectl Plugin](https://github.com/rook/kubectl-rook-ceph).
```

Comment:
good to add issue https://github.com/rook/kubectl-rook-ceph/issues/74  for tracking?

Reply:
Until someone is committed to that complex feature, I'll suggest to not include it in the feature list for a specific release.